app/design/frontend/Magento/blank/web/css/source/_layout.less

```diff
@@ -125,14 +125,11 @@
     }
 
     .page-layout-2columns-left {
-        .main {
-            padding-left: @layout-column__additional-sidebar-offset
-        }
-
         .sidebar-additional {
             clear: left;
             float: left;
             padding-left: 0;
+            padding-right: @layout-column__additional-sidebar-offset;
         }
     }
 
```

Comment:
reduce double padding spaces ! Previously magento accidentally add spaces from both `main` and `sidebar`